partitura/io/musescore.py

```diff
@@ -153,7 +153,7 @@ def render_musescore(part, fmt, out_fn=None, dpi=90):
 
     """
     mscore_exec = find_musescore3()
-
+    print(mscore_exec)
```

Comment:
same here, do we need this print?

Reply:
not that I know. I didn't change this file, this is probably because of a mistake in a rebase.

Reply:
reverted in a new commit.